### PR TITLE
Upgrade scRNA-seq workflows from 0.4 to 0.5

### PR DESCRIPTION
**Pull request by lldelisle**
I did it manually because the CI autoupdate was not able to autoupdate them.
I also incorporated the changes in test syntax identified in #530

### CODE REVIEW

**Comment by lldelisle** [comment]
Ready for review.

File: workflows/scRNAseq/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-cellplex-tests.yml [hunk 1]
```diff
@@ -54,15 +54,11 @@
...
-        - that: "has_text_matching"
-          expression: ">subsample</th><td class=\"data-coloured mqc-generalstats-star-uniquely_mapped_percent \"><div class=\"wrapper\"><span class=\"bar\""
-        - that: "has_text_matching"
-          expression: "></span><span class=\"val\">3[0-9].[0-9]%</span></div></td><td class=\"data-coloured mqc-generalstats-star-uniquely_mapped \"><div class=\"wrapper\"><span class=\"bar\""
+        has_text_matching:
+          expression: "<td class=\"data-coloured mqc_generalstats_star_uniquely_mapped_percent_1 \"><div class=\"wrapper\"><span class=\"bar\"></span><span class=\"val\">33.[0-9]<span class=\"mqc_small_space\"></span>%</span></div></td>"
     Seurat input for gene expression (filtered):
-      class: Collection
-      collection_type: list:list
+      attributes: { collection_type: list:list }
```
Comment (bgruening) [inline workflows/scRNAseq/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-cellplex-tests.yml:61]: I liked the old syntax more.

**Review by bgruening — APPROVED** [review]

**Comment by mvdbeek** [comment]
> I did it manually because the CI autoupdate was not able to autoupdate them.

What was the error there ?

**Comment by lldelisle** [comment]
```
Traceback (most recent call last):
  File "/home/ldelisle/planemo/bin/planemo", line 8, in <module>
    sys.exit(planemo())
             ^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/decorators.py", line 92, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/planemo/cli.py", line 103, in handle_blended_options
    return f(*args, **kwds)
           ^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/planemo/commands/cmd_autoupdate.py", line 142, in cli
    updated_workflow = autoupdate.autoupdate_wf(ctx, config, workflow)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/planemo/autoupdate.py", line 364, in autoupdate_wf
    _update_wf(config, workflow_id)
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/planemo/autoupdate.py", line 287, in _update_wf
    config.user_gi.workflows.refactor_workflow(wf["id"], actions=[{"action_type": "upgrade_all_steps"}])
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/bioblend/galaxy/workflows/__init__.py", line 701, in refactor_workflow
    return self._put(payload=payload, url=url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/bioblend/galaxy/client.py", line 190, in _put
    return self.gi.make_put_request(url, payload=payload, params=params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ldelisle/planemo/lib/python3.11/site-packages/bioblend/galaxyclient.py", line 260, in make_put_request
    raise ConnectionError(
bioblend.ConnectionError: Unexpected HTTP status code: 400: {"err_msg":"Workflow step has upgrade messages","err_code":0,"err_data":{"5":{"solo|wasp_conditional|waspOutputMode":"No value found for 'Actiavte WASP filtering'. Using default: ''."}}}
```

**Pull request merged**